apps/dalmatiner_db/src/dalmatiner_tcp.erl

```diff
@@ -76,7 +75,20 @@ loop(Socket, Transport, State) ->
                                         dict = bkt_dict:new(Bucket,
                                                             State#state.n,
                                                             State#state.w)},
+                                {incomplete, <<>>});
+                {stream, Bucket, Delay, Resolution} ->
+                    lager:info("[tcp] Entering stream mode for bucket '~s' "
+                               "with max delay: ~p and resolution: ~pms",
+                               [Bucket, Delay, Resolution]),
+                    ok = dalmatiner_opt:set_resolution(Bucket, Resolution),
```

Comment:
I think we should print at least add a lager error to when the resolution couldn't be set, if not returning an error to the client.

It might be very confusing if the client keeps the connection killed w/o any hint of why.